### PR TITLE
Enhanced Recurring Donations Nightly Batch Job performance test flow

### PR DESCRIPTION
**Pull request by force2b**
# Critical Changes

# Changes

# Issues Closed

# Community Ideas Delivered

# New Metadata

# Deleted Metadata

### CODE REVIEW

**Review by davidmreed — APPROVED** [review]
Approving for CODEOWNERS, defer to @prescod on the effect of the changes.

**Comment by force2b** [comment]
@prescod Is anything else needed for this?

**Review by prescod — APPROVED** [review]

**Pull request merged**
